recipe/meta.yaml

```
@@ -33,49 +36,55 @@ requirements:
     - chameleon
     - execnet
     - isodate
-    - jplephem
     - lxml
     - netcdf4
     - hdf4
     - pillow >=5,<6
-    - python-dateutil
     - pytz
-    - pyqt
-    - qt
+    - pyqt >=5,<6
+    - qt >=5,<6
     - requests
     - scipy
-    - sgp4
     - skyfield >=1.12
-    - six
     - tk
     - owslib
     - unicodecsv
     - fs_filepicker
-    - future
     - cftime >=1.0.1
     - matplotlib ==3.0.2
     - proj4 <6
     - flask
     - flask-httpauth
     - werkzeug
+    - flask-socketio
+    - flask-sqlalchemy
+    - passlib
+    - gitpython
+    - psycopg2
+    - validate_email
+    - multidict
+    - pint
 
 test:
```

I think the `gitpython` package doesn't depend on `git` to allow people to have external installations. You can add it as a test dependency to fix the current failure:
```suggestion
test:
  requires:
    - git
```

---

added git to the common requirements. I will discuss that in the gitpython-feedstock later.

I revoked the version tag, we fix first the main() problem but it looks now on a good path